src/ros_thread.cpp

```diff
@@ -87,10 +87,8 @@ void RosThread::startRos()
 
   nh_ = rclcpp::Node::make_shared(name.str());
 
-  rclcpp::QoS qos(rclcpp::QoSInitialization::from_rmw(rmw_qos_profile_system_default));
-
   rosout_sub_ = nh_->create_subscription<rcl_interfaces::msg::Log>(
-    "/rosout", qos, std::bind(&RosThread::emptyLogQueue, this, std::placeholders::_1));
+    "/rosout", rclcpp::QoS(rclcpp::KeepLast(10)).transient_local().reliable(), std::bind(&RosThread::emptyLogQueue, this, std::placeholders::_1));
```

Comment:
Would it make more sense to replace `rmw_qos_profile_system_default` with `rcl_qos_profile_rosout_default` in the initialization of qos?

Reply:
If that's an importable thing then sure!

Reply:
I'm checking in Foxy, but it appears to build in Humble.

Reply:
Done!